website/src/arcade/ArcadeSuccessView.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+
+type ArcadeSuccessViewProps = {
+  data: unknown;
```

Comment:
Should we type this stronger to something serializable like `Record<string, string | number | boolean | null>` or is that just being silly?